CMakeLists.txt

```diff
@@ -271,9 +271,20 @@ endif()
 # If it's preexisting, leave it alone.
 include(InstallHelpers)
 
+# For $CMAKE_INSTALL_{DATAROOT,MAN, ...}DIR
+include(GNUInstallDirs)
+
+file(GLOB MANPAGES
+  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
+  man/nvim*.1)
+
+install_helper(
+  FILES ${MANPAGES}
+  DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
+
 install_helper(
   DIRECTORY runtime
-  DESTINATION share/nvim)
+  DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/nvim")
```

Comment:
You don't need to add the quotes here.  CMake is smart, and even if the variables value has spaces, it will do the right thing.